src/deck-options.md

```diff
@@ -4,260 +4,219 @@
 
-- v1 applies parent limits to children, regardless of which deck you click on
-- v2 behaves similarly to v1 for new cards. For reviews, only the limits of
-  the deck you click on are honored.
-- v3 honors the limits of the deck you click on, and any decks inside it.
-  Limits from parents above the deck you clicked on are ignored.
-
-For more information, please see the [v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits) page.
+For earlier versions, see [this FAQ page](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html).
 
 Studying new cards will temporarily increase the number of reviews you need to
-do a day, as freshly learnt material needs to be repeated a number of times
+do a day, as newly learnt material needs to be repeated a number of times
 before the delay between repetitions can increase appreciably. If you are
 consistently learning 20 new cards a day, you can expect your daily reviews to
 be roughly about 200 cards/day. You can decrease the reviews required by
-introducing fewer new cards each day, or by turning off new card display until
-your review burden decreases. More than one Anki user has excitedly studied
-hundreds of new cards over their first few days of using the program, and then
+introducing fewer new cards each day until your review burden decreases. More than one Anki user has excitedly studied
+hundreds of new cards over their first few days of using the program, and then has
 become overwhelmed by the reviews required.
 
 ### Maximum Reviews/Day
 
-Allows you to set an upper limit on the number of reviews to show each day.
+Allows you to set an upper limit on the number of review cards to show each day.
 When this limit is reached, Anki will not show any more review cards for the
-day, even if there are some waiting. If you study consistently, this setting can
+day, even if there are more waiting. If you study consistently, this setting can
 help to smooth out occasional peaks in due card counts, and can save you from a
-heart attack when returning to Anki after a week off. When reviews have been
+heart attack when returning to Anki after taking a week off. When reviews have been
 hidden due to this option, a message will appear in the congratulations screen,
 suggesting you consider increasing the limit if you have time.
 
-In [the v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html#daily-limits) and
-v1 schedulers, the counts are affected by parents/selected decks in the same way
-as new cards.
+When studying a deck that has subdecks inside it, the review limit behaves similarly to the new card limit.
 
-In the v2 scheduler, the limit is taken solely from the deck you select - any
-limits on its parents or child decks are ignored.
+Anki includes any learning cards that have [crossed the day boundary](https://docs.ankiweb.net/preferences.html#scheduler) (interday learning cards) in the review count,
```

Comment:
Since this is part of the manual, we should use a local link (like ./preferences.md) instead of a remote link.